rosdep/python.yaml

```diff
@@ -1642,6 +1642,19 @@ python-flask-restful:
         packages: [flask-restful]
     xenial: [python-flask-restful]
     yakkety: [python-flask-restful]
+python-flask-socketio:
```

Comment:
@tiffca Flask-SocketIO docs suggest it is a Python 3 package only (which means the key should be `python3-flask-socketio` instead) and I can see it listed in platform-specific package managers (which means that the key should be `python3-flask-socketio-pip` instead, though it'd best to not use those instead of pip if possible).